extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -169,13 +172,17 @@ private function renderItem( $type, array $data ) {
 
 		if ( $type === 'image' ) {
 			$images = array();
+
 			for ( $i = 0; $i < count($data); $i++ ) {
 				$data[$i][ 'context' ] = self::MEDIA_CONTEXT_INFOBOX;
 				$data[$i] = $helper->extendImageData( $data[$i] );
+				$data[$i] = $this->nodeSanitizer->sanitizeInfoboxFields( $type, $data[$i] );
```

Comment:
We could you provide object here instead of running sanitization inside. It seems more clear to me that way.